qiskit_dynamics/pulse/__init__.py

```diff
@@ -19,18 +19,17 @@
 
 .. currentmodule:: qiskit_dynamics.pulse
 
-This module contains tools to interface qiskit pulse with qiskit dynamics. Indeed,
-Qiskit-dynamics simulates time evolution using the :class:`Signal` class. However,
-Qiskit pulse specifies pulse instructions using a schedule. This module contains
-the required converters to convert Qiskit pulse schedules into signals such that
-they can be simulated.
+This module contains tools to interface qiskit pulse with qiskit dynamics. Indeed, Qiskit Dynamics
+simulates time evolution using the :class:`Signal` class. However, Qiskit pulse specifies pulse
+instructions using a schedule. This module contains the required converters to convert Qiskit pulse
+schedules into signals such that they can be simulated.
 
 Converters
 ==========
 
 The conversion from a pulse schedule to a list of signals is done with the
-:class:`InstructionToSignals` converter. The example below shows a schedule and
-the resulting signals converted using
+:class:`InstructionToSignals` converter. The example below shows a schedule and the resulting
+signals converted using
```

Comment:
I might put a few words between the codeblock and the jupyter execute to implicitly indicate that these two lines of code don't produce a plot, e.g. "which results in a schedule such as the following"

Reply:
Changed this.